README.md

```diff
@@ -16,10 +16,9 @@
 
 <td>
 <table>
-<tr><th></th>                 <th>Release</th> <th>Development</th> </tr>
-<tr><td> Documentation </td>  <td> (coming soon) </td> <td> <a href="https://docs.zeek.org/projects/spicy">Spicy Manual</a> </td></tr>
-<tr><td> Changelog </td>      <td> (coming soon) </td> <td> <a href="/CHANGES">CHANGES</a> </td></tr>
-<tr><td> Build status </td>   <td> (coming soon) </td> <td> <a href="https://cirrus-ci.com/github/zeek/spicy/main"><img src="https://api.cirrus-ci.com/github/zeek/spicy.svg" alt="Build status"> </a> </td></tr>
+<tr><td>Documentation</td><td><a href="https://docs.zeek.org/projects/spicy">Spicy Manual</a> </td></tr>
+<tr><td>Changelog</td>    <td><a href="/CHANGES">CHANGES</a> </td></tr>
+<tr><td>Build status</td> <td><a href="https://cirrus-ci.com/github/zeek/spicy/main"><img src="https://api.cirrus-ci.com/github/zeek/spicy.svg" alt="Build status"> </a> </td></tr>
```

Comment:
I was thinking we'd leave both columns in here, so that people see the information for both current release and dev. Except I guess for the docs, where we don't have two versions, can just span that across the two columns.

Reply:
I cannot find a documented or undocumented way to get the Cirrus CI build status svg for a tag (branch would be possible, but that's not the same as the release), https://cirrus-ci.org/guide/writing-tasks/#embedded-badges.

I could add some text for a tag's status here (e.g., `Cirrus CI status`). Like I mentioned offline, I am not sure this is too useful as tags are immutable.

Reply:
Let's use the release branch for the status then? Not exactly the same but close enough I'd say. 

The alternative is to do get rid of this table altogether and replace it with something else showing the "Docs / CHANGES / build status" information.

(Not a release blocker either way of course)

Reply:
So this would be my suggestion 
```
<td>
<table>
<tr><th></th>              <th>Release</th> <th>Development</th> </tr>
<tr><td>Documentation</td> <td colspan="2" align="center"><a href="https://docs.zeek.org/projects/spicy">Spicy Manual</a></td></tr>
<tr><td>Changelog</td>     <td><a href="https://github.com/zeek/spicy/blob/v1.0.0/CHANGES">CHANGES</td> <td><a href="/CHANGES">CHANGES</a></td></tr>
<tr>
  <td>Build status</td>
  <td><a href="https://cirrus-ci.com/github/zeek/spicy/release"><img src="https://api.cirrus-ci.com/github/zeek/spicy.svg?branch=release" alt="Build status release"></a></td> 
  <td><a href="https://cirrus-ci.com/github/zeek/spicy/main">   <img src="https://api.cirrus-ci.com/github/zeek/spicy.svg" alt="Build status development"></a></td>
</tr>
</table>
</td>
```